release/models/system/openconfig-system-management.yang

```diff
@@ -127,6 +127,77 @@ module openconfig-system-management {
     }
   }
 
+
+  grouping system-netconf-config {
+    description
+      "Configuration data for the netconf ptotocol";
+
+    leaf ssh {
```

Comment:
Should this be an enumeration named "netconf-transport" to define the transport in use? This currently assumes the client is using SSHv2.

Reply:
Agree, while SSH transport is the current defacto transport for NETCONF, it is not the only one and some implementations can support NETCONF directly over TCP transport, local socket, etc..

Reply:
incoparated . please review